### DISCUSSION TITLE
Rename Fossify Dialer to Fossify Phone?

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#538] [Polls]
Please also comment your reasons below.

**Comment by inson1** [comment]
@naveensingh Wouldnt phone name also mean uniting dialer and contacts into one app?

**Reply by naveensingh** [reply]
Not necessarily. Google has a "Google Phone" and a "Google Contacts" app. But... now that I think about it, uniting them may not be a bad idea.

To me, Dialer sounds like something primarily focused on outgoing calls and that's not the only thing Fossify Dialer does.

**Reply by inson1** [reply]
@naveensingh some call it for some reason feature, idk why, I would like one app tbh

You are right, I forgot about Google Contacts

**Comment by inson1** [comment]
For me no, I dont see the point

Plus I dont like Phone naming tbh.

**Comment by GitCurious** [comment]
I find there's so much crossover between the dialer and the contacts, I would be happy if they could be merged into one and just use the one app instead of two.

No preference about name but voted No anyway.

**Comment by inson1** [comment]
https://www.reddit.com/r/Fossify/comments/1918zoe/rename_fossify_dialer_to_fossify_phone/

**Comment by naveensingh** [comment]
Ok, I'm renaming Dialer to Fossify Phone.

**Reply by vertigo220** [reply]
The main problem with this is discoverability. For years, I've periodically searched F-Droid and the Play Store for a new dialer, as I had some years ago that were great then became abandoned, and none since have been very good. Searching right now for "dialer" doesn't show this app, and while I may have eventually found it by searching phone (I ended up finding it from reading about Fossify in a user review on a Simple app), it's definitely possible users will overlook it because of this. This should be easily remedied by simply using the word dialer in the description so it shows up in searches for that term, but it's not used once in the entire thing for some reason. So regardless of what it's name, it should have the other term used there to make it more visible.
